### ISSUE TITLE
Keyboard trapped when tabbing through standalone mapping application 

### ISSUE THREAD

**Issue by dan-bowerman** [fgpv-vpgf/fgpv-vpgf#1111]
When accessing the viewer through the index-one.html page (full screen, no template), the application gets stuck. This can be replicated by tabbing into the application until the pop-up appears instructing how to give keyboard focus to the map. You cannot escape the pop-up, and the tab must be closed and re-opened in order to use the map again.
![image](https://cloud.githubusercontent.com/assets/5294596/17941115/e7a3d98a-69ff-11e6-801a-7fe65808b46e.png)

This does not appear to be an issue when using a template like WET.

![](https://files.gitter.im/fgpv-vpgf/wbHc/Animation.gif)

**Comment by AleksueiR** [comment]
Related: #834
